### ISSUE TITLE
Feature request: Configurable drawer with onclick listener

### ISSUE THREAD

**Issue by canmrt** [Telerik-Verified-Plugins/NativePageTransitions#32]
An option for setting drawer width as a percentage to screen width or as px.
A callback which is triggered after clicking the screenshot.

Those two options would be great. Setting a fixed right margin is not flexible on tablet layouts. Also beeing able to detect a tap on the screenshot provides more opportunities for the interaction.

**Comment by trevor403** [comment]
This feature actually isn't required. The area under the screenshot is clickable, therefore you can handle this yourself. I used a div to measure out what the area of the screenshot on the side of the screen was. Then I just listend for touch events in the bounding box and called the drawer close animation at that point. Very simple!

**Comment by canmrt** [comment]
I think it's utterly hacky and unwise to use invisible overlays and modify DOM elements instead of providing a simple event handling by the plug-in.

**Comment by trevor403** [comment]
This is true, it would be best add an event handler to the screenshot view from the plugin side. However it means that you would have to make the plugin object eventful. This would require some very simple javascript to add event listeners to the nativepagetransitions object. This can be done other ways like adding a new plugin method that has a Cordova callback id for the click event.
